src/ui/components/ide/monaco-editor.tsx

```diff
@@ -111,29 +128,49 @@ export default function (props: { onChange: OnChange }) {
 	}, [activeFilePath]);
 
 	const activeFile = getActiveFile();
+
 	return (
-		<Editor
-			theme="theme"
-			height="100%"
-			width="100%"
-			language="typescript"
-			value={activeFile?.content}
-			path={activeFile?.fullPath || 'empty.ts'}
-			options={{
-				fontSize: 18,
-				fontWeight: '500',
-				fontFamily: 'Fira Mono',
-				minimap: {
-					enabled: false,
-				},
-				overviewRulerLanes: 0,
-				lineNumbersMinChars: 2,
-			}}
-			beforeMount={(monaco) => {
-				setMonacoRef(monaco);
-			}}
-			onMount={setEditor}
-			onChange={props.onChange}
-		/>
+		<>
+			<div style={{ position: 'absolute', zIndex: 1000 }}>
```

Comment:
We use https://v6.mantine.dev/, so let's not re-invent the wheel here. Check out how other components are maintained, like https://github.com/metz-sh/simulacrum/blob/main/src/ui/components/pretty-paper/pretty-paper.component.tsx